packages/frontend/core/src/commands/affine-creation.tsx

```diff
@@ -53,6 +60,10 @@ export function registerAffineCreationCommands({
       icon: <PlusIcon />,
       label: t['com.affine.cmdk.affine.new-workspace'](),
       run() {
+        track.$.cmdk.creation.createWorkspace({
+          control: 'newWorkspace',
```

Comment:
don't have to specify control, the segment/module matrix can identify this event already